src/reducers/clientActions.js

```diff
@@ -0,0 +1,39 @@
+// Action Types
+export const SET_USER = "SET_USER";
+export const SET_ROLES = "SET_ROLES";
+export const SET_THEME = "SET_THEME";
+export const SET_LANGUAGE = "SET_LANGUAGE";
+
+const initialState = {
```

Comment:
initialState içinde bazı değerler (örneğin, user ve theme) null veya undefined olarak başlatılabilir. Bu, özellikle obje yapıları değişkenlik gösterecekse daha güvenli olabilir